### DISCUSSION TITLE
Weird zoom/jumping behaviour without any interaction

### DISCUSSION THREAD

**Discussion by fakob** [pixijs/pixijs#7285] [Q&A]
Hi, 

I am not sure if this is related to pixi, or is just a webgl thing. Thought I just throw it out there and see if this looks familiar to anyone.

### Current Behavior
My canvas has these random jumps where it enlarges the viewport and then randomly jumps back again. In the zoomed in state I can not click any of my elements, which might actually point to the fact that it has nothing to do with pixi, but might be a webgl issue? Also because it is only happening on one of my Macs, but there it seem to happen on all Chromium based browsers. My app doesn't run on the other ones so I could not check.

https://user-images.githubusercontent.com/4619772/110684173-1da46280-81dd-11eb-9e76-b0c766cedb4a.mp4

Has anyone seen that behaviour before?

### Possible Solution
Use a different computer :-)

### Environment
- **`pixi.js` version**: 5.3.8
- **Browser & Version**: Chrome Version 89.0.4389.82 
- **OS & Version**: macOS Mojave Version 10.14.6
- **Running Example**: sorry can not share an example yet

**Comment by bigtimebuddy** [comment]
I converted this to a discussion, since your post was not a specific bug in PixiJS. If you are able to narrow down the cause, please post an issue. Otherwise, a discussion a good place to get feedback or help debugging problems in projects.

**Reply by fakob** [reply]
Makes sense. Thanks @bigtimebuddy, I am still not fully used to the discussion feature :-)

**Comment by CyanLetter** [comment]
Wanted to chime in and second the idea that this may be hardware / browser / OS specific, and unrelated to PixiJS. I am seeing the exact same behavior on a late 2013 iMac. In my case, the environment is:

- `pixi.js` version: 6.0.2
- Browser & Version: Chrome Version 90.0.4430.93
- OS & Version: macOS Mojave Version 10.14.6
- GPU: NVIDIA GeForce GTX 775M

As you stated, this occurs for me in Chromium browsers only. I do not see the same behavior in Firefox. 
I also do *not* see this behavior on other macs (2017 MacBook Air, macOS Big Sur 11.3.1, Intel HD Graphics 6000).

Also not the first time I have encountered WebGL issues on this machine. Chrome has at times blacklisted this particular GPU on macOS, and while that does not appear to be the case here, may be evidence of instability on certain hardware configurations. 

Not sure if this warrants further investigation, but for now I'm going to go with your "use a different computer" solution.

**Reply by fakob** [reply]
Thanks for your comment/confirmation @CyanLetter and sorry to hear that you have the same problem.
